.github/workflows/examples-java-codeql-analysis.yml

```
@@ -0,0 +1,43 @@
+name: "CodeQL Analysis"
+
+on:
+  push:
```

Although that's tecnically valid for GitHub actions, it confuses me when reading the YAML as I'm expecting there to be an empty object:

```suggestion
  push: {}
```